apps/portals/standards/src/config/resources.ts

```diff
@@ -1 +1,25 @@
-export const dataSql = `SELECT * FROM syn65676531`
+const DST_TABLE_ID = 'syn65676531'
+// can replace with specific version if wanted
+
+// for the Explore page table:
+export const dataSql = `
+    SELECT
+        concat('[', acronym, '](/Explore/Standard/DetailsPage?id=', id, ')') as acronym,
+            name, category, collections, topic as topics, relevantOrgAcronym as organizations, isOpen, registration FROM ${DST_TABLE_ID}
```

Comment:
FYI @katiestahl and @Sigfried , it looks like including the `topic` column causes the page row limit to significantly decrease (presumably due to the possible max row size, so the Synapse backend is limiting the response page row count).